test/e2e/storage/drivers/in_tree.go

```diff
@@ -1104,23 +1104,33 @@ var _ testsuites.InlineVolumeTestDriver = &gcePdDriver{}
 var _ testsuites.PreprovisionedPVTestDriver = &gcePdDriver{}
 var _ testsuites.DynamicPVTestDriver = &gcePdDriver{}
 
-// InitGceDriver returns gcePdDriver that implements TestDriver interface
+// InitGcePdDriver returns gcePdDriver that implements TestDriver interface
 func InitGcePdDriver() testsuites.TestDriver {
+	var supportedTypes sets.String
+	var capFsGroup bool
+	if framework.NodeOSDistroIs("windows") {
+		supportedTypes = sets.NewString("ntfs")
+		capFsGroup = false
+	} else {
+		supportedTypes = sets.NewString(
+			"", // Default fsType
+			"ext2",
+			"ext3",
+			"ext4",
+			"xfs",
+			"ntfs",
```

Comment:
we don't want this for linux?

Reply:
fixed

Reply:
ntfs is still here, we should remove it right?